qubes/__init__.py

```diff
@@ -299,6 +299,11 @@ def __repr__(self):
     def __hash__(self):
         return hash(self.__name__)
 
+    def __lt__(self, other):
+        if isinstance(other, property):
+            return self.__name__ < other.__name__
+        else:
+            return self < other
```

Comment:
So, maybe instead of `sorted(..., key=)` in `property_list()`, this operator should be defined (using the algorithm from `property_list()`)?